experimental/packages/api-logs/src/types/AnyValue.ts

```diff
@@ -14,16 +14,29 @@
  * limitations under the License.
  */
 
-import { AttributeValue } from '@opentelemetry/api';
+export type AnyValueScalar = string | number | boolean;
+
+export type AnyValueArray = Array<AnyValue>;
 
 /**
  * AnyValueMap is a map from string to AnyValue (attribute value or a nested map)
  */
 export interface AnyValueMap {
-  [attributeKey: string]: AnyValue | undefined;
+  [attributeKey: string]: AnyValue;
 }
 
 /**
- * AnyValue is a either an attribute value or a map of AnyValue(s)
+ * AnyValue can be one of the following:
+ * - a scalar value
+ * - a byte array
+ * - array of any value
+ * - map from string to any value
+ * - empty value
  */
-export type AnyValue = AttributeValue | AnyValueMap;
```

Comment:
reviewer note: Since `AttributeValue` is:

```
export type AttributeValue =
  | string
  | number
  | boolean
  | Array<null | undefined | string>
  | Array<null | undefined | number>
  | Array<null | undefined | boolean>;
```

Then removing it and adding the `AnyValueScalar` and `AnyValueArray` should already include all the previous allowed type, thus this type only extends the previous one and thus I don't think it's a breaking change